public/app/features/explore/state/utils.ts

```diff
@@ -73,11 +77,11 @@ export const makeExplorePaneState = (): ExploreItemState => ({
     from: null,
     to: null,
     raw: DEFAULT_RANGE,
-  } as any,
+  } as unknown as TimeRange,
```

Comment:
Why can't we use an actual `TimeRange` instance here? Because this is not a test. Same for `AbsoluteTimeRange`.

Reply:
I added this fix thinking it is a test file. I removed this changes in https://github.com/grafana/grafana/pull/61298/commits/ff905e3773a715b1548971d203d163389cd1d873